jenkins/get-text.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x
+
+env | sort
+echo
+
+# The docker container test script will know to copy out
+echo "Using workspace: $WORKSPACE"
+mkdir -p $WORKSPACE/artifacts/
+
+# make selinux happy via http://stackoverflow.com/a/24334000
+chcon -Rt svirt_sandbox_file_t $WORKSPACE//artifacts/
+
+# Run the gettext to extract strings for translation from source java files
+./docker/test -p -c "cp-test -g -c ${CHANGE_BRANCH}" -n "${STAGE_NAME}-${BUILD_TAG}"
```

Comment:
There is no need to use docker containers to do this work. The reason we have the containers because they are set up with running tomcat/postgres and candlepin installed already in order to run spec tests etc. This task (gettext & msgattrib) though, does not require a running candlepin/tomcat/postgres. Instead, since you have already checked out candlepin, you could just do the work here on the checkout (without running containers).

Also, this task is supposed to be done for specific branches: master and the 2-3 latest HOTFIX branches that we support and have significant changes going in to them (at this time, this is candlepin-3.1-HOTFIX and candlepin-2.9-HOTFIX). So that would be: run gettext and msgattrib for master, commit the changes, then checkout candlepin-3.1-HOTFIX and repeat the process, then checkout candlepin-2.9-HOTFIX and repeat again.

We could have a static list of those branches (at this moment with 3 values) that we'd have to update over time as more hotfix branches are created.